mtp_noms_ops/apps/security/forms/object_list.py

```diff
@@ -146,6 +158,29 @@ def get_query_data(self, allow_parameter_manipulation=True):
         return query_data
 
 
+class SendersFormV2(BaseSendersForm):
+    """
+    Search Form for Senders V2.
+    """
+    search = forms.CharField(
+        label=_('Search payment source name or email address'),
+        required=False,
+        help_text=_('Common or incomplete names may show many results'),
+    )
+
+    # NB: ensure that these templates are HTML-safe
+    filtered_description_template = 'Results containing {filter_description}, ordered by {ordering_description}.'
+    unfiltered_description_template = 'Payment sources, ordered by {ordering_description}'
+
+    description_templates = (
+        ('prisoner number or name "{search}"',),
```

Comment:
i think this ought to be `“{search}”` for consistency with other quotation marks

Reply:
👍 you mean `‘{search}’` I guess.

Reply:
Ok `“{search}”` is the version that the team is happy with so I'll go with that.